Sources/BraveWallet/Crypto/Stores/NFTDetailStore.swift

```diff
@@ -48,26 +54,43 @@ struct NFTMetadata: Codable, Equatable {
   }
 }
 
+struct NFTAttribute: Codable, Equatable, Identifiable {
+  var type: String
+  var value: String
+  var id: String { type }
+  
+  enum CodingKeys: String, CodingKey {
+    case type = "trait_type"
+    case value
+  }
+}
+
 class NFTDetailStore: ObservableObject, WalletObserverStore {
+  private let assetManager: WalletUserAssetManagerType
   private let rpcService: BraveWalletJsonRpcService
   private let ipfsApi: IpfsAPI
-  let nft: BraveWallet.BlockchainToken
+  let owner: BraveWallet.AccountInfo?
```

Comment:
updated [883a39f](https://github.com/brave/brave-ios/pull/8408/commits/883a39feda833d08a48a172d39d10dc4d0f63cb7) 
only fetching balance when `Hide Unowned` is on in `NFTStore`.
fetching NFT balance in NFT Details if `owner` is nil -> this will bring the owner account info when user comes from asset search.